staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/validating/dispatcher.go

```diff
@@ -238,13 +240,184 @@ func (d *validatingDispatcher) Dispatch(ctx context.Context, attr admission.Attr
 	}
 	if len(errs) > 1 {
 		for i := 1; i < len(errs); i++ {
-			// TODO: merge status errors; until then, just return the first one.
 			utilruntime.HandleError(errs[i])
```

Comment:
This is intended to log any error that is surfaced to the client.

I think we want to move this call from here, to where we decide that we are not surfacing an error, i.e., somewhere in the `aggregateWebhookErrors` function.